lib/rules.js

```diff
@@ -77,10 +77,29 @@ LiveRules.prototype.remove = function (rules, cb) {
     }
   });
 };
+/**
+ * delete rules by ids
+ * @param {Array<string>} rules_ids
+ * @param cb
+ */
+LiveRules.prototype.removeByIds = function (rules_ids, cb) {
+  var json = {
+    rule_ids: rules_ids
+  };
+  request.post({
+    url: this._api + '?_method=delete',
+    json: json,
+    headers: { 'Authorization': this._auth }
+  }, function (err, response, body) {
+    if (err) cb(err);
+    else if (response.statusCode >= 200 && response.statusCode < 300) cb(null, body);
+    else cb(new Error('Unable to delete rules. Request failed with status code: ' + response.statusCode + 'body: '+ JSON.stringify(body)));
+  });
+};
```

Comment:
Can you extract this logic so we can reuse it in the method above? It is the same, except the payload is different. After this change, I can merge and release a new minor version later today,.